dev.geopjr.Tuba.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: dev.geopjr.Tuba
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
+command: dev.geopjr.Tuba
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - /share/vala
+  - '*.la'
+  - '*.a'
+modules:
+  - name: libsoup-2.4
```

Comment:
you should really port to libsoup 3.

Reply:
Tracked https://github.com/GeopJr/Tuba/issues/12